src/Trustline.ts

```diff
@@ -41,16 +41,20 @@ export class Trustline {
   }
 
   /**
-   * Prepares an ethereum transaction object for creating a trustline update request. Called by initiator
-   * of update request.
+   * Prepares an ethereum transaction object for creating a trustline update request.
+   * Called by initiator of update request.
    * @param networkAddress Address of a currency network.
    * @param counterpartyAddress Address of counterparty who receives trustline update request.
-   * @param creditlineGiven Proposed creditline limit given by iniator to counterparty,
-   *              i.e. 1.23 if network has to 2 decimals.
-   * @param creditlineReceived Proposed creditline limit received by iniator from counterparty,
-   *                 i.e. 1.23 if network has to 2 decimals.
-   * @param options Options for creating an ethereum transaction. See type `TLOptions` for more information.
-   * @param options.decimals Decimals of currency network can be provided manually if known.
+   * @param creditlineGiven Proposed creditline limit given by initiator to counterparty,
+   *                        i.e. 1.23 if network has to 2 decimals.
+   * @param creditlineReceived Proposed creditline limit received by initiator from counterparty,
+   *                           i.e. 1.23 if network has to 2 decimals.
+   * @param options Options for creating an `updateTrustline` ethereum transaction.
+   *                See type `TrustlineUpdateOptions` for more information.
+   * @param options.interestRateGiven Proposed interest rate given by initiator to counterparty in % per year.
```

Comment:
What is the reason to add this to the option object instead of having the interests as an own argument along with creditlineGiven? I think it would be cleaner as a normal argument with a default value.

Reply:
Here I also thought about this. So you mean something like
```javascript
public async prepareUpdate(
  networkAddress,
  counterpartyAddress,
  clGiven,
  clReceived,
  interestGiven = 0,
  interestReceived = 0,
  options = {}
) {...}
```
The problem would be then that someone could not call something like
```javascript
prepareUpdate(
  '0x...',
  '0x..',
  1000,
  2000,
  {
    // options
    ...
  }
)
```

Reply:
I think someone should anyway never do the second options, as it is confusing what is going on and the proper way with naming the argument should still work:
```
prepareUpdate(
  '0x...',
  '0x..',
  1000,
  2000,
  options: {
    // options
    ...
  }
)
```
Or is that not possible in javascript?

Reply:
I am afraid that named arguments like this is not possible in Javascript...

Reply:
Ah okay, then I guess this was the right decision if we want to have interests optional.